tgui-next/packages/tgui/interfaces/Achievements.js

```diff
@@ -15,23 +60,43 @@ export const Achievements = props => {
           <Box as="Table">
             {data.achievements
               .filter(x => x.category === category)
-              .map(achievement => (
-                <tr key={achievement.name}>
-                  <td style={{'padding': '6px'}}>
-                    <Box className={achievement.icon_class} />
-                  </td>
-                  <td style={{'vertical-align': 'top'}}>
-                    <h1>{achievement.name}</h1>
-                    {achievement.desc}
-                    <Box
-                      color={achievement.achieved ? "good" : "bad"}
-                      content={achievement.achieved ? "Unlocked" : "Locked"} />
-                  </td>
-                </tr>
-              ))}
+              .map(achievement => {
+                if (achievement.score)
+                {
+                  return (<Score
+                    name={achievement.name}
+                    desc={achievement.desc}
+                    icon_class={achievement.icon_class}
+                    value={achievement.value} />);
+                }
+                else
+                {
+                  return (<Achievement
+                    name={achievement.name}
+                    desc={achievement.desc}
+                    icon_class={achievement.icon_class}
+                    value={achievement.value} />);
+                }
+              })}
           </Box>
         </Tabs.Tab>
       ))}
+      <Tabs.Tab
+        label={"High Scores"}>
+        {data.highscore.map(highscore => {
+          return (
+            <Section key={highscore.name} title={highscore.name}>
+              <LabeledList>
+                {
+                  Object.keys(highscore.scores).map(key =>
+                  {
+                    return (<LabeledListItem key={key} label={key}>{highscore.scores[key]}</LabeledListItem>);
+                  })
+                }
```

Comment:
You know, it could've been 3 lines of code, you made it 7.

Not only that, you also run off the 80 character mark. Structure it vertically.